src/Google.Cloud.Functions.Hosting/Extensions/FunctionsFrameworkLoggingExtensions.cs

```diff
@@ -35,8 +35,8 @@ public static ILoggingBuilder AddFunctionsConsoleLogging(this ILoggingBuilder bu
         {
```

Comment:
nit: Line 28 mentions "single line per log entry". Might not be the best place to go into a lengthy explanation, but maybe a note about scopes and exceptions might be good.

Reply:
Yup, will do.